src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,21 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../Types/todo';
```

Comment:
[CODE STYLE] - According to the checklist, we should prepare data in one place (App component) and pass it to child components. Avoid using `import` for data in non-root components. Instead, you should pass the `todo` object with its associated `user` already attached to it from the parent component.